db/db_impl_write.cc

```diff
@@ -348,7 +348,7 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
       versions_->SetLastSequence(last_sequence);
     }
     MemTableInsertStatusCheck(w.status);
-    write_thread_.ExitAsBatchGroupLeader(write_group, w.status);
+    write_thread_.ExitAsBatchGroupLeader(write_group, status);
```

Comment:
Should we apply the same change to this line too? https://github.com/facebook/rocksdb/pull/3201/files#diff-7850efea42ee1acae3ac06f1633842f7R580